src/Core/src/Platform/iOS/ContentView.cs

```diff
@@ -103,8 +112,24 @@ void SetClip()
 			// we use the double to get the correct value. Here, again, we use the double to get the correct clip shape size values.
 			var strokeWidth = 2 * strokeThickness;
 
-			var bounds = new RectF(0, 0, (float)frame.Width - strokeWidth, (float)frame.Height - strokeWidth);
+			var bounds = new RectF(strokeWidth / 2, strokeWidth / 2, (float)frame.Width - strokeWidth, (float)frame.Height - strokeWidth);
```

Comment:
Can you add a comment here explain what's happening to the bounds ?

For example why isn't the Width `(float)frame.Width - (strokeWidth /2)` ? And why are we clipping just half of the stroke?!